template/5-mvp.md

```diff
@@ -1,34 +1,71 @@
 
-*User stories about how various personas will use the product in context.*
+### User Story Exchange Student
 
-*Identify and prioritise the key features required.*
+1. As an exchange student, I want to be able to take part in EPFL public events so that I can integrate more easily into my new environment.
 
-*Justify the importance of each feature.*
+### User Story Repeater Student
+
+1. As a repeater student, I want to be able to get to know my new classmates so that I have people to take classes with. 
+
+### User Story All EPFL Students
+
+1.	As an EPFL student, I want to be able to join events to pursue my passions.
+2.	As an EPFL student, I want to be able to meet new people who share the same passion as me so that I can make new friends
+
+### User Story Association
+
+1.	As an association, I want to be able to promote my events in order to have more participants.
+
+## Key Features
+
+The main aim of Echo is to bring EPFL students together. That's why our application allows them to organise events around their passions, bringing together students from the same class or section. The key to our success lies in the number of events actually organised. Key features of our app include:
+* Event participation - This is the most obvious feature, offering our users the chance to discover personalised events based on their passions and curiosity. They can easily find them thanks to a powerful search tool and profile customisation.
+* Getting to know people from the same background - Coming to EPFL can be difficult, especially if you don't know anyone. Echo is keen to encourage people from the same class or section to get to know each other, so students can organise events restricted to a certain milieu. 
+* Associations - At present, the associations are by far the most active in creating events on campus. However, they all have the same problem: a lack of participants. Echo enables associations to promote their events to users and reach their target audience more easily.
 
 ## Success Criteria
 
-*How will you evaluate the success of the MVP?*
+### Event Creation
 
-*Metrics include user penetration, quality / satisfaction.*
+To assess the success of Echo, we will be looking at the number of events officially held. We estimate that an event brings together an average of 10 people. Our application will be a success if we manage to organise at least 1 event per week for every 100 students. So, for a first-year computer science class of 600 students, at least 6 events should be organised within this class using our application. 
 
-*If applicable, progress in discussions with ecosystem partners / investors / customers.*
+### User Adoption & Engagement
 
-## Features Outside the Scope
+* In its first month at the start of the academic year, our application plans to reach at least 30% of new EPFL students (first-year, exchange and repeat students ~ 1,000 users), and at least 10% of the rest of the student body (~ 1,000 users). 
```

Comment:
Also number of user signups to the app.
User growth rate: monitoring the increase in users over a certain period: helps to know the popularity of the app among the user population.

Reply:
That's quiet similar to what i'm saying. I simply follow the example given for pocket campus